justfile

```diff
@@ -10,11 +10,12 @@ cuda_510_driver := "510.47.03"
 cuda_470_driver := "470.82.01"
 cuda_515_driver := "515.65.01"
 cuda_535_driver := "535.154.05"
```

Comment:
can you bump 535 to the latest swell, in case we need to quickly revert AB back to 535.

Reply:
https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-161-08/index.html

Reply:
Updated

Reply:
Looks like corresponding fabric manager version was not published for 535 -first time I'm seeing this: https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/

Reply:
Splitting 535 update to a separate PR, in case we want to revert it later: https://github.com/Azure/aks-gpu/pull/33